sdk/core/Azure.Core/src/Shared/OperationHelpers.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Azure
+{
+    internal static class OperationHelpers
+    {
+        public static TimeSpan DefaultPollingInterval { get; } = TimeSpan.FromSeconds(1);
+
+        public static T GetValue<T>(ref T? value) where T: class
+        {
+            if (value == null)
```

Comment:
Should is `is null` with generic parameter in case it overrides and doesn't properly handle `Equals(null)`.